src/styles/blocks/head-block/icon.scss

```diff
@@ -0,0 +1,7 @@
+.icon {
```

Comment:
The icon element has an absolute position but there is no parent element with a relative position. In order to position the icon relative to a parent element, the parent element should have a position of relative. Without a parent with relative position, the icon will position itself relative to the screen which can cause issues with different screen sizes.